reviews/tests.py

```diff
@@ -362,21 +382,21 @@ def test_auto_review_observation(self):
         reasons = auto_review_observation(o)
         self.assertEqual(len(reasons), 2)
 
-    def test_auto_review_task(self):
+    def test_auto_review_registration_age_min(self):
         self.study.has_sessions = True
-        self.study.age_groups.set(AgeGroup.objects.filter(pk=4))  # adolescents
+        self.study.age_groups.set(self.adolescents)
         self.study.save()
 
         reasons = auto_review(self.proposal)
-        self.assertEqual(len(reasons), 0)
+        self.assertEqual(len(reasons), 1)  # adolescents are minors
 
         s1 = Session.objects.create(study=self.study, order=1)
         s1_t1 = Task.objects.create(session=s1, order=1)
-        s1_t1.registrations.set(
-            Registration.objects.filter(pk=6)
-        )  # psychofysiological measurements
+        psychofysiological_measurement = Registration.objects.filter(pk=6)
```

Comment:
Um, I would rather we match by name of the registration here. At least if the name changes, the test will be guaranteed to fail. 

If the pk changes, that could go unnoticed for a long time.